libs/domain/search/src/services/suggestion/renderer/suggestion.renderer.ts

```diff
@@ -0,0 +1,102 @@
+import { Product, ProductMediaContainerSize } from '@spryker-oryx/product';
+import { SemanticLinkType } from '@spryker-oryx/site';
+import { i18n } from '@spryker-oryx/utilities';
+import { html, TemplateResult } from 'lit';
+import { when } from 'lit/directives/when.js';
+import { SuggestionLinks } from '../../../models';
+import { SuggestionRenderer } from './suggestion-renderer.service';
+
+export const productSuggestionRenderer: SuggestionRenderer<Product[]> = (
+  products,
+  params
+) => {
+  const renderProductsSection = (query?: string): TemplateResult => {
+    return html`
+      <section>
+        <h5>${i18n('search.box.products')}</h5>
+        ${products?.map(renderProduct)}
+
+        <oryx-button
+          outline
+          @click=${(event: Event) =>
+            event.target?.dispatchEvent(
+              new CustomEvent('oryx.close', { bubbles: true, composed: true })
+            )}
+        >
+          <oryx-content-link
+            .options=${{
+              type: SemanticLinkType.ProductList,
+              params: { q: query },
+            }}
+            .content=${{ text: i18n('search.box.view-all-products') }}
+          ></oryx-content-link>
+        </oryx-button>
+      </section>
+    `;
+  };
+  const renderProduct = (product: Product): TemplateResult => {
```

Comment:
Wondering about separating them out, and move product renderer to product lib?

Reply:
Basically this renderer already changed @tobi-or-not-tobi in another PR

Reply:
I'm still wondering if we can improve this area going forward, but happy to first let this land and unblock us with the (urgently needed) new UI. we have a cleaner picture then and can further refine.